web/server.js

```
@@ -14,8 +15,12 @@ module.exports = function(config) {
       coinbase_secret: config.coinbase_secret
     }
   });
+  server.connection({
+    host: config.host,
+    port: config.port
+  });
 
-  server.pack.register(require("hapi-auth-bearer-token"), function(err) {});
+  server.register(require("hapi-auth-bearer-token"), function(err) {});
```

Are we just discarding that error for no reason?